pycodestyle.py

```diff
@@ -1665,8 +1682,8 @@ def parse_udiff(diff, patterns=None, parent='.'):
             if path[:2] in ('b/', 'w/', 'i/'):
                 path = path[2:]
             rv[path] = set()
-    return dict([(os.path.join(parent, path), rows)
-                 for (path, rows) in rv.items()
+    return dict([(os.path.join(parent, child), rows)
```

Comment:
Why are you changing this here?

Reply:
I run flake8 and found this error... so  I went for this drive by... is 1 line :)

I have reverted this.